susemanager-utils/testing/automation/VERSION.SUSE-Manager

```diff
@@ -8,3 +8,5 @@ PUSH2OBS_CONTAINER=devel/galaxy/manager/head/docker/containers/suma-push-to-obs
 # This value is is used if the file for the PR is not found (we assume we are on a branch)
 IDEMPOTENCY_SCHEMA_BASE_VERSION='4.1.8'
 IDEMPOTENCY_REPORTDB_SCHEMA_BASE_VERSION='4.3.1'
+SCHEMA_PACKAGES='susemanager-schema-4.1.8-1.2.uyuni.noarch.rpm'
+REPORTDB_SCHEMA_PACKAGES='uyuni-reportdb-schema-4.3.1-1.3.uyuni1.noarch.rpm susemanager-schema-4.3.8-1.3.uyuni1.noarch.rpm susemanager-schema-utility-4.3.8-1.3.uyuni1.noarch.rpm'
```

Comment:
I adapted the head containers to bring them on par with the uyuni containers. I also used the packages from Beta1.
These are now the names:

```suggestion
SCHEMA_PACKAGES='susemanager-schema-4.2.4-1.3.noarch.rpm'
REPORTDB_SCHEMA_PACKAGES='susemanager-schema-4.3.8-150400.1.5.noarch.rpm susemanager-schema-utility-4.3.8-150400.1.5.noarch.rpm uyuni-reportdb-schema-4.3.1-150400.1.5.noarch.rpm'
```